### PR TITLE
Added distribution image variables to digitalocean recipes/modules

### PR DESCRIPTION
**Pull request by VltraHeaven**
`var.droplet_image` variable added to digitalocean recipe and module. This allows the user to set which Linux distribution will be used on RKE1 cluster nodes.

### CODE REVIEW

File: modules/infra/digitalocean/main.tf [hunk 1]
```diff
@@ -47,6 +47,10 @@ resource "digitalocean_droplet" "droplet" {
       "echo 'Completed cloud-init!'"
     ])
   }
+
+  lifecycle {
```
[the remark this answers is not in the document]
Comment (dkeightley) [reply modules/infra/digitalocean/main.tf:51]: Hi @VltraHeaven, what do you think about the above? It's a quick change to implement, I can help with it if needed also

[the remark this answers is not in the document]
Comment (dkeightley) [reply modules/infra/digitalocean/main.tf:51]: As I can't push to your forked repo, are you also able to rebase off the main branch of this repo? Just need to [pull in this commit](https://github.com/rancher/tf-rancher-up/commit/baafa585c6a9da482e782f54f6bf38b6828fd50c) to fix the CI checks.

[the remark this answers is not in the document]
Comment (VltraHeaven) [reply modules/infra/digitalocean/main.tf:51]: Hey @dkeightley 
I think your changes look good. I can make the adjustments and rebase my fork so it merges properly. Thank you for looking at this for me.

[the remark this answers is not in the document]
Comment (VltraHeaven) [reply modules/infra/digitalocean/main.tf:50]: Made the requested changes - https://github.com/rancher/tf-rancher-up/commit/a1402e88ef6938cf616f76cc7342ef6cdaf5a584

**Review by dkeightley — APPROVED** [review]

**Pull request merged**